### ISSUE TITLE
Color output

### ISSUE THREAD

**Issue by nichtich** [LibreCat/Catmandu#232]
Color output on command line would be nice. I implemented CSV and JSON in color for [wdq](https://metacpan.org/pod/wdq) but it's a hack.

**Comment by vpeil** [comment]
Yes, color output would be cool. There are several tools to improve the CLI experience:

- https://metacpan.org/pod/Term::ANSIColor

- https://metacpan.org/pod/CLI::Helpers

**Comment by netsensei** [comment]
We use Term::ANSIColor in the [Datahub::Factory](https://github.com/thedatahub/Datahub-Factory) project we maintain. (which actually runs on top of Catmandu libraries).

We implemented a quick, basic -v CLI option which allows us to do verbose output on STDOUT. Like status messages during complex processing. These messages are coloured.

See: https://github.com/thedatahub/Datahub-Factory/blob/master/lib/Datahub/Factory/Flash.pm

We don't colour log4perl log messages, because we use these primarily for logging to a custom file appender. But we could colour those given the possibility that log4perl also outputs to STDOUT if we configure it to do so.
